appveyor.yml

```diff
@@ -2,3 +2,25 @@ configuration: Release
 test:
   assemblies:
     - Argu.Tests.dll
+
+install:
+  - ps: $env:CLI_VERSION = "1.0.0-preview2-003121"
+  - ps: $env:DOTNET_INSTALL_DIR = "$pwd\.dotnetcli"
+  # Download install script to install .NET cli in .dotnet dir

```

Comment:
.net core sdk preview2 is already installed in an appveyor build worker image, no need to install it.
you need to add `os: Visual Studio 2015` to use that image (it's ok for dev, has pratically everything for .net dev)

btw a better version of appveyor is [there](https://github.com/dotnet/netcorecli-fsc/blob/master/appveyor.yml#L12-L20) or in [dotnet docs](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/using-ci-with-cli), but use preview1 atm